config/setupMPI.cmake

```diff
@@ -399,7 +399,12 @@ macro( setupSpectrumMPI )
       # 1 resource set; OMP_NUM_THREADS tasks; no gpu; no binding
       set( MPIEXEC_OMP_PREFLAGS "--nrs 1 -c $ENV{OMP_NUM_THREADS} -g 0 --bind none")
     elseif( MPIEXEC_EXECUTABLE MATCHES lrun )
-      set( MPIEXEC_OMP_PREFLAGS "-c $ENV{OMP_NUM_THREADS} --threads=$ENV{OMP_NUM_THREADS} --bind=off -v" )
+      if( DEFINED ENV{OMP_NUM_THREADS} )
+        # --bind=off
+        set( MPIEXEC_OMP_PREFLAGS "--pack --threads=$ENV{OMP_NUM_THREADS} -c$ENV{OMP_NUM_THREADS} -v" )
```

Comment:
It seems odd to have the same number of cores and threads, how does lrun interpret this?

Reply:
The `-v` option will print the `jsrun` command.  I tested these types of commands and they looked correct.  
```
lrun -n 2 --pack --threads=1 -v js_task_info # no OpenMP
lrun -n 2 --pack --threads=10 -c 10 -v js_task_info  # with 10 OpenMP threads
```
Can you repeat this test and advise me?  This appeared to do what I wanted, but maybe I misunderstood the output.

Reply:
Looking at the jsrun visualizer I think these commands are doing the right thing. The "threads" command seems to just be for OpenMP and not "threads per core".